### PR TITLE
Fix #1544: Add java 8 changes to the changelog.

### LINKED ISSUE #1544: Switching to a supported Java version
We are currently still supporting Oracle Java 1.6 which is not supported by Oracle anymore. Even Java 1.7 is not supported anymore. See http://www.oracle.com/technetwork/java/eol-135779.html

We should switch to Java 8 rather soon. For this we need to know if any/how many of our customers still depend on older Java versions.

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+* Java 8 or higher is needed to run Marathon, since Java 6 and 7 support has reached end of life. 
 
 * `--revive_offers_for_new_apps` is now the default. If you want to avoid resetting filters
   if new tasks need to be started, you can disable this by `--disable_revive_offers_for_new_apps`.
 
 ### Overview
 
+#### Marathon uses Java 8 
+
+Java 8 is out for over a year now. The support for older Java versions have reached end of life.
```
Comment (gkleiman) [inline changelog.md]: Java 8 has been out for more than a year now. The support for older versions of Java has reached end of life.

File: changelog.md [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
   if new tasks need to be started, you can disable this by `--disable_revive_offers_for_new_apps`.
 
 ### Overview
 
+#### Marathon uses Java 8 
+
+Java 8 is out for over a year now. The support for older Java versions have reached end of life. 
+We switched completely to the latest stable JVM release.
+We compile, test and run Marathon against Java 8.
+Note for the Marathon docker image: the base image of Marathon is now the standard java:8-jdk docker image.
```
Comment (gkleiman) [inline changelog.md]: s/docker/Docker/g

**Comment by gkleiman** [comment]
LGTM, added just a few suggestions.

**Comment by aquamatthias** [comment]
Thanks for the feedback!

**Comment by gkleiman** [comment]
Can you please rebase the commit?

**Pull request merged**